deployment/cloudways.rst

```diff
@@ -0,0 +1,158 @@
+appropriate value according to your location.
+
+.. image:: /_images/deployment/cloudways/server-timezone-1.png
+   :alt: Select server timezone
+
+Step 5: SSH Terminal
+--------------------
+
+After changing server timezone, in **Server Management** click on **Launch SSH
+Terminal** under  **Master Credentials** where you can find your login
+credentials for SSH Terminal.
+
+.. image:: /_images/deployment/cloudways/master-credentials.png
+   :alt: Cloudways Master Credentials
+
+Login to SSH with your Master Credentials.
+
+.. image:: /_images/deployment/cloudways/symfony-7.png
+   :alt: login to SSH
+
+Since the application folder contains a ``index.php`` fie by default, remove it
+from your ``public_html/`` directory executing the ``rm -Rf *`` command.
+
+Step 6: Symfony Installation
+----------------------------
+
+Now in your ``public_html/`` directory, run the following command to install
+Symfony 3 on your server:
+
+.. code-block:: terminal
+
+    $ composer create-project symfony/framework-standard-edition your_project_name "3.1.*"
+
+This command will create a Symfony project in ``public_html/``. You can also
+change the version number to install any other Symfony version.
+
+Step 7: DB Information and Finishing Project Installation
+---------------------------------------------------------
+
+Now Symfony will start downloading to your server. During downloading, it will
+ask for your database information.
+
+.. image:: /_images/deployment/cloudways/db-info.png
+   :alt: database information
+
+You can find out your DB information under **Application Management** go to
+**Access detail** and see MySQL Access.
+
+.. image:: /_images/deployment/cloudways/mysql-access.png
+   :alt: database information
+
+After providing database information, you will see the following screen which
+means that  Symfony has been installed successfully.
+
+.. image:: /_images/deployment/cloudways/symfony-installed.png
+   :alt: Final Symfony Installation
+
+Step 7: Running in Browser
+--------------------------
+
+After a successful installation, in **Application Management** go to **Access
+detail** and copy your Application URL.
+
+.. image:: /_images/deployment/cloudways/mysql-access.png
+   :alt: database information
+
+Open your favorite web browser and access to the "Symfony Welcome page" using
+the following URL: http://your_application_url/your_project_directory/web
+
+This is how the Symfony Welcome page should look like:
+
+.. image:: /_images/deployment/cloudways/symfony-home.png
+   :alt: symfony homepage
+
+Finally your Symfony project is ready for development. 
+
+Deploying From Github 
+======================
+
+You can also deploy Symfony application from github. copy the HTTP URL of the repository,
+
+.. image:: /_images/deployment/cloudways/http_url.png
+   :alt: symfony homepage
+
+Open SSH terminal on cloudways Using Master Credentials as shown in Step 5. Move to your project folder,
+`cd application/{your app folder}/public_html`
```

Comment:
I would reword the sentence to "[...] Move to your project folder and run the clone command there:" and then show both commands in a `terminal` code block:

```rst
[...] Move to your project folder and run the clone command there:

.. code-block:: terminal

    cd application/{your app folder}/public_html
    git clone https://github.com/username/your-repository.git
```